### DISCUSSION TITLE
Unable to get Github app installation id for org

### DISCUSSION THREAD

**Discussion by jlorezz** [prisma/prisma#12169] [Q&A] [answered]
Hey i am getting an issue with cloud.prisma.io when trying to get my prisma db. 

`We were unable to create your project: Unable to get Github app installation id for org: ... `

![image](https://user-images.githubusercontent.com/29343633/156889377-9f7f4d95-88cb-4c21-935a-28c190ab1171.png)

**Comment by nurul3101** [comment, marked as answer]
Hey @florianbress 👋 

Can you start an Intercom conversation on [Prisma Data Platform](https://cloud.prisma.io/)?

Intercom would be the ideal channel for anything related to Prisma Data Platform.

**Reply by jlorezz** [reply]
How do I do that?

**Reply by nurul3101** [reply]
You should be able to see an intercom icon once you login on the lower right side of the page.

Like this:
<img width="266" alt="Screenshot 2022-03-07 at 3 52 27 PM" src="https://user-images.githubusercontent.com/13049676/157013012-eca8b9be-e5bf-47ac-955d-7450916de020.png">

**Reply by nurul3101** [reply]
Update: This issue occurs if the GitHub handle name is changed after signing up with Prisma Data Platform.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
